### PR TITLE
Update network-config-with-networkd.md to show way to see all IP addresses on an interface

### PR DESCRIPTION
**Pull request by krishjainx**
# Update network-config-with-networkd.md to show way to see all IP addresses on an interface

Updated the network-config-with-networkd.md file to include instructions on how to view all IP addresses assigned to a network interface. This addition will help users verify the success of their configuration.

### CODE REVIEW

File: docs/setup/customization/network-config-with-networkd.md [hunk 1]
```diff
@@ -164,6 +164,19 @@ storage:
...
+$ ip -4 addr show dev ens33
+3: ens33: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc fq_codel state UP group default qlen 1000
+    altname enp2s1
+    inet 10.0.1.2/24 brd 10.0.1.255 scope global ens33
+       valid_lft forever preferred_lft forever
+    inet 10.0.1.3/24 brd 10.0.1.255 scope global secondary ens33
+       valid_lft forever preferred_lft forever
+ ```
+By executing the command ip -4 addr show dev ens33, you can obtain detailed information about the ens33 interface. The output includes the interface's state, such as whether it is UP or DOWN, its assigned IP addresses, the corresponding subnet masks, and other relevant details. In the provided example, the ens33 interface has two assigned IP addresses, 10.0.1.2 and 10.0.1.3, both belonging to the 10.0.1.0/24 subnet.
+
```
Comment (jepio) [inline docs/setup/customization/network-config-with-networkd.md:179]: Wouldn't it be more useful to provide output that matches the example above? So eth0 with the two ips on line 161 and 163.

Comment (krishjainx) [reply docs/setup/customization/network-config-with-networkd.md]: I think this should be OK @jepio

File: docs/setup/customization/network-config-with-networkd.md [hunk 2]
```diff
@@ -164,6 +164,18 @@ storage:
           Gateway=10.0.1.1
 ```
 
+To verify whether your configuration was successful and view all IP addresses associated with a specific interface, you can use the following syntax: ip -(4 or 6) addr show dev interface_name. Here is an example of the command and its output:
```
Comment (jepio) [inline docs/setup/customization/network-config-with-networkd.md:167]:
```suggestion
To verify whether your configuration was successful and view all IP addresses associated with a specific interface, you can use the following syntax: `ip [-4|-6] addr show dev <interface_name>`. Here is an example of the command and its output:
```

File: docs/setup/customization/network-config-with-networkd.md [hunk 3]
```diff
@@ -164,6 +164,18 @@ storage:
...
+
+```
+$ ip -4 addr show dev eth0
+3: eth0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc fq_codel state UP group default qlen 1000
+    inet 10.0.0.101/24 brd 10.0.0.255 scope global eth0
+       valid_lft forever preferred_lft forever
+    inet 10.0.1.101/24 brd 10.0.1.255 scope global secondary eth0
+       valid_lft forever preferred_lft forever
+ ```
+By executing the command ip -4 addr show dev eth0, you can obtain detailed information about the eth0 interface. The output includes the interface's state, such as whether it is UP or DOWN, its assigned IP addresses, the corresponding subnet masks, and other relevant details.
```
Comment (jepio) [inline docs/setup/customization/network-config-with-networkd.md:177]:
```suggestion
By executing the command `ip -4 addr show dev <interface_name>`, you can obtain detailed information about the eth0 interface. The output includes the interface's state, such as whether it is UP or DOWN, its assigned IP addresses, the corresponding subnet masks, and other relevant details.
```

**Review by jepio — APPROVED** [review]

**Review by jepio — APPROVED** [review]

**Pull request merged**